src/components/ButtonMenu.stories.tsx

```diff
@@ -222,3 +226,68 @@ export function IconMenuWtihSearchableList() {
 
   return <ButtonMenu defaultOpen trigger={{ icon: "archive" }} items={menuItems} searchable />;
 }
+
+export function GlobalNavLinkMenu() {
+  const menuItems: MenuItem[] = [
+    { label: "Page action", onClick: action("Test item clicked") },
+    { label: "Internal Link", onClick: "/fakeRoute" },
+    { label: "External Link - Homebound.com", onClick: "https://www.homebound.com" },
+  ];
+
+  return (
+    <div css={Css.mlPx(200).$}>
+      <h2 css={Css.lg.$}>Global Nav Link Menu</h2>
+      <ButtonMenu
+        trigger={{ navLabel: "Menu trigger", variant: "global" }}
+        items={menuItems}
+        persistentItems={[
+          { label: "Persistent Action", onClick: action("Persistent action clicked") },
+          { label: "Destructive Action", onClick: action("Destructive Action Clicked"), destructive: true },
+        ]}
+        defaultOpen
+        contrast={true}
+      />
+    </div>
+  );
+}
+
+export function MenuWithContrast() {
+  const menuItems: MenuItem[] = [
+    { label: "Page action", onClick: action("Test item clicked") },
+    { label: "Internal Link", onClick: "/fakeRoute" },
+    { label: "External Link - Homebound.com", onClick: "https://www.homebound.com" },
+  ];
+
+  return (
+    <>
+      <div css={Css.df.fdr.$}>
+        <div>
+          <h2 css={Css.lg.$}>Default - Without Contrast</h2>
+          <ButtonMenu
+            trigger={{ label: "Menu trigger" }}
+            items={menuItems}
+            persistentItems={[
+              { label: "Persistent Action", onClick: action("Persistent action clicked") },
+              { label: "Destructive Action", onClick: action("Destructive Action Clicked"), destructive: true },
+            ]}
+            defaultOpen
+          />
+        </div>
+
+        <div css={Css.mlPx(200).$}>
+          <h2 css={Css.lg.$}>With Contrast</h2>
+          <ButtonMenu
+            trigger={{ label: "Menu trigger", variant: "tertiary" }}
+            items={menuItems}
+            persistentItems={[
+              { label: "Persistent Action", onClick: action("Persistent action clicked") },
+              { label: "Destructive Action", onClick: action("Destructive Action Clicked"), destructive: true },
+            ]}
+            defaultOpen
+            contrast={true}
+          />
+        </div>
```

Comment:
@bdow wondering if I should remove this story component: `MenuWithContrast` since we plan on reverting the  [`tertiary` style changes](https://github.com/homebound-team/beam/commit/bbc5225b146fda30018d6489e8f9bb0353e90a2a)

Reply:
Yup, makes sense. I think we still want to add another Story for a "NavLink Button Menu"

Reply:
@bdow i added one here https://github.com/homebound-team/beam/pull/762/files#diff-b7cfab5b5cb43b275fde9ba97abe97424d1a938e8c9437f1a1cdaa33020a36c1R230
but i think the `NavLink Button Menu` is a better name, ill update to that